tasks-by-users/Rehan-Khan-Engineer/09-Typewriter-animation/style.css

```diff
@@ -0,0 +1,87 @@
+:root {
+  --bg-color: rgb(214, 122, 16);
+  --type-writer-speed: 6s;
+  --type-writer-steps: 24;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
Add these properties to universal selector @Rehan-Khan-Engineer

Reply:
added CSS reset to universal selector as a good practice. @Abbas4tech @mmusk